dts/arm/nordic/nrf52840.dtsi

```diff
@@ -137,6 +137,22 @@
 			interrupt-names = "wdt";
 			label = "WDT";
 		};
+
+		cryptocell: crypto@5002a000 {
+			compatible = "nordic,nrf-cc310";
+			reg = <0x5002A000 0x1000>;
+			label = "CRYPTOCELL";
+			status = "ok";
+			#address-cells = <1>;
+			#size-cells = <1>;
+			cc310: crypto@5002b000 {
+				compatible = "arm,cryptocell-310";
+				reg = <0x5002B000 0x1000>;
```

Comment:
Is this address (0x5002B000) and the respective range, kind of "random"?

Reply:
No, both address and range reflects actual location and size of CC310 registers.